data/mllib/sample_hierarchical_data.csv

```diff
@@ -0,0 +1,150 @@
+5.1,3.5,1.4,0.2
```

Comment:
Minor point - this wouldn't really be CSV though. I imagine the example shows parsing a common encoding like this on purpose.